src/javascripts/components/navigation.js

```diff
@@ -136,7 +136,13 @@ Navigation.prototype.init = function () {
 
   if (typeof window.matchMedia === 'function') {
     this.mql = window.matchMedia('(min-width: 40.0625em)')
-    this.mql.addEventListener('change', this.setHiddenStates.bind(this))
+
+    // IE and Safari < 14 do not support MediaQueryList.addEventListener
+    if (this.mql.hasOwnProperty('addEventListener')) {
```

Comment:
As discussed on slack, recommend using `in` instead of `hasOwnProperty` as `mediaQueryList` inherits `addEventListener` so `hasOwnProperty` won't ever find it.

```suggestion
    if ('addEventListener' in this.mql) {
```